### PR TITLE
creating script to automate creation of additional storage accounts.

### PR DESCRIPTION
**Pull request by haydonryan**
Creation of additional storage accounts is frustrating to do by hand. Created a script to create additional storage accounts.

Note they are created as LRS accounts.

### CODE REVIEW

File: scripts/createAdditionalStorageAccounts.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      exit 0
+fi
+
+#Get main storage account key
+echo Getting Main Storage Key....
+
+mainkey=$(azure storage account keys list --resource-group $1 $2 | grep --line-buffered Primary | awk -F " " '{print $3}')
+echo $mainkey
+#Create table if not exists
+azure storage table create --table stemcells --account-name $2 --account-key $mainkey
```
Comment (AbelHu) [inline scripts/createAdditionalStorageAccounts.sh]: Thanks for your commit. Are you upgrading from preview2? If you are using the latest Azure template, the table will be created by default. If you create it manually, you also should create two containers stemcell and bosh for the main storage account. And you should set the permission of the container stemcell to blob.

azure storage container create --account-name $2 --account-key $newkey bosh
azure storage container create --account-name $2 --account-key $newkey --permission Blob stemcell

Set permission if the permission is not set when the container was created
azure storage container set --account-name $2 --account-key $mainkey --permission Blob --container stemcell

**Comment by haydonryan** [comment]
I've made the changes as well as updated the documentation App Gateway. Merge away.

**Comment by bingosummer** [comment]
@haydonryan 
Sorry for late response.

Because CPI now supports creating additional storage accounts ([#77](https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/pull/77)), there is no need to add a script to create storage accounts.

Could you please close this PR and submit a new one which only contains "improving explaination of dns setup requirements for websockets on AG" and "adding additional known issue that Application Gateway"?

Thanks for your contributions.

**Comment by AbelHu** [comment]
As Bin said as above, close this PR.

**Pull request closed without merging**